js-rattler/tsconfig.rollup.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Maybe we can use [extends](https://www.typescriptlang.org/tsconfig/#extends) to remove some duplication here?

Reply:
done